### PR TITLE
Add donate link

### LINKED ISSUE #4173: Add donate link to sidebar
BTCPay Server Foundation can now accept donations from individuals through OpenSats. This is big for two reasons:
- Finally individual users can donate
- Entities can also donate through OpenSats and get tax deduction if they're US entity.

For this reason, it would be nice if we could add a link or a button somewhere (sidebar?) that can guide people through process of donating? A simple thing would just be linking to [btcpayserver.org/donate](https://btcpayserver.org/donate/) where we could have a step by step donation process (Individual vs Corporate donation pop up or something). Thoughts? A tracking issue for UX on btcpayserver.org [here](https://github.com/btcpayserver/btcpayserver.org/issues/173)

We can potentially utilize "what's new" section for 1.7.0 or minor release in between to notify people of this new possiblity.

_Issue created from a [Mattermost message](https://chat.btcpayserver.org/_redirect/pl/f5mbnhm38b8zi8xqyrwxbet8na) *by @pavlenex*._

### PR DESCRIPTION
**Pull request by dennisreimann**
Closes #4173.

grafik

grafik

### CODE REVIEW

**Review by dstrukt — APPROVED** [review]
It's gettin' busy down there - in time we might need to do something else, but for now it looks great on all the breakpoints!

tACK.

**Comment by pavlenex** [comment]
Too wordy, can we have something simpler [View all supporters] [💚Donate] This still requires us to improve donation page on the website since right now it's a mess. https://btcpayserver.org/donate/

We need to add foundation and open-sats and remove individual contributors, since individual donations should go through open-sats.

**Comment by dstrukt** [comment]
ooof, yeah the donate page definitely needs some love

@pavlenex good point, could be reduced, but my 2sats, as an alternative .. the icon in the text line might be a little messier than just 

`View all supporters or Donate`

e.g.
Screen Shot 2022-10-26 at 12 43 52 AM

**Comment by pavlenex** [comment]
Agreed with what's proposed by @dstrukt just `donate` is enough.

File: BTCPayServer/Views/Shared/_BTCPaySupporters.cshtml [hunk 1]
```diff
@@ -62,5 +62,6 @@
     </div>
 </div>
 <p class="text-center">
-    <a href="https://foundation.btcpayserver.org" target="_blank" rel="noreferrer noopener">View all supporters</a>
+    <a href="https://foundation.btcpayserver.org" target="_blank" rel="noreferrer noopener">View all supporters</a> or
+    <a href="https://btcpayserver.org/donate/" target="_blank" rel="noreferrer noopener">donate to the project</a>
```
Comment (dennisreimann) [inline BTCPayServer/Views/Shared/_BTCPaySupporters.cshtml:66]:
```suggestion
    <a href="https://btcpayserver.org/donate/" target="_blank" rel="noreferrer noopener">Donate</a>
```

**Review by pavlenex — APPROVED** [review]
